src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -305,6 +305,7 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 	<_AndroidSequencePointsMode Condition=" '$(_AndroidSequencePointsMode)' == ''">None</_AndroidSequencePointsMode>
 	<_InstantRunEnabled Condition=" '$(_InstantRunEnabled)' == '' ">False</_InstantRunEnabled>
 	<_AndroidBuildPropertiesCache>$(IntermediateOutputPath)build.props</_AndroidBuildPropertiesCache>
+	<_AndroidDesignTimeBuildPropertiesCache>$(IntermediateOutputPath)designtimebuild.props</_AndroidDesignTimeBuildPropertiesCache>
```

Comment:
I know its just a file however I would place this in the `designtime` folder like the other files. This keeps it completely isolated.

Reply:
The problem I was having was that `$(_AndroidDesignTimeResDirIntermediate)` was defined later, on line.

I can probably move things around to get this to work.